### DISCUSSION TITLE
Vec3 property is not RGB color version in Blender3.0

### DISCUSSION THREAD

**Discussion by HSDSZ** [bnpr/Malt#131] [Feedback & Feature Requests]
I don't know whether this is a bug or not
In blender 3.0 alpha version

vec3 and vec4 property no longer appear as a color, but a vector.
![Bug](https://user-images.githubusercontent.com/36077492/133831141-dccda740-1a2c-4306-81c7-9f67efc74945.PNG)

Any plan to solve this?

**Comment by pragma37** [comment]
> Any plan to solve this?

Blender 3.0 is planned to be released in December, so they still have plenty of time to make breaking changes.
So sure, 3.0 will be supported at release. But I probably won't make the update until late November, once their changes are settled and documented.

**Reply by JPLost** [reply]
This seems to have been caused by rBc99cb814520480379276192f044c673ef857447b, where a few socket names had been updated and few defaults changed. 

https://developer.blender.org/rBc99cb814520480379276192f044c673ef857447b

**Comment by pragma37** [comment]
Just fixed this in b327ee641995460d4398a099aa7048e7ed590a7d :)

**Reply by JPLost** [reply]
Can confirm that rgb slider works correctly on 3.1.0 alpha.
